src/test/java/nc/students/ayaz/controllers/VideoControllerTest.java

```diff
@@ -98,7 +98,7 @@ public void shouldReturnVideoWhenPostRequestSentFirstAndThenGetRequestSent() thr
 
         assertEquals(HttpStatus.OK, response.getStatusCode());
         Video fetchedVideo = mapper.readValue(response.getBody(), Video.class);
-        assertTrue(fetchedVideo.equals(new Video("FunnyCats")));
+        assertEquals(fetchedVideo, new Video("FunnyCats"));
```

Comment:
&#x1F34E; **Issue**
If you inspect assertEquals in detail you'll see that its actually the first argument that should be "the expected result" and "the actual result" belongs to the second argument.
When your test will fail, it will make the difference. Junit will show you what it expected and what it got. And in the current implementation such a message won't make much sense, cause it will read something like that:
"expected 'null' but got Video('FunnyCats') instead". You can try it yourself. Its just a nice thing to be aware of.